### ISSUE TITLE
Wrong Swedish translation

### ISSUE THREAD

**Issue by vespucci-reporter** [simonpoole/beautified-JOSM-preset#354]
Description:
---

The covered passageway Arcade have been wrongly translated into "video-game parlor" in Swedish.

Device info:
---
<table>
<tr><td>App version</td><td>18.1.0.0</td></tr>
<tr><td>App version code</td><td>2302</td></tr>
<tr><td>Android build version</td><td>G986BXXUEFVH3</td></tr>
<tr><td>Android release version</td><td>12</td></tr>
<tr><td>Android SDK version</td><td>31</td></tr>
<tr><td>Android build ID</td><td>SP1A.210812.016.G986BXXUEFVH3</td></tr>
<tr><td>Device brand</td><td>samsung</td></tr>
<tr><td>Device manufacturer</td><td>samsung</td></tr>
<tr><td>Device name</td><td>y2s</td></tr>
<tr><td>Device model</td><td>SM-G986B</td></tr>
<tr><td>Device product name</td><td>y2sxeea</td></tr>
<tr><td>Device hardware name</td><td>exynos990</td></tr>
<tr><td>ABIs</td><td>[arm64-v8a, armeabi-v7a, armeabi]</td></tr>
<tr><td>ABIs (32bit)</td><td>[armeabi-v7a, armeabi]</td></tr>
<tr><td>ABIs (64bit)</td><td>[arm64-v8a]</td></tr>
</table>

Extra info:
---
<table>
<tr><td>OSM display name</td><td><A href="https://openstreetmap.org/user/Christoffre"/>Christoffre</A></td></tr>
</table>

**Comment by simonpoole** [comment]
Yes this needs a translation context so that the two cases can be differentiated.
